lib/krb5/fcache.c

```diff
@@ -249,7 +249,7 @@ scrub_file (int fd)
 KRB5_LIB_FUNCTION krb5_error_code KRB5_LIB_CALL
 _krb5_erase_file(krb5_context context, const char *filename)
 {
-    int fd;
+    int fd = -1;
```

Comment:
This change is unnecessary.   There is no use of 'fd' before it is assigned a value as the output of open() on line 260.